webapp/src/contexts/CommonsContext.tsx

```diff
@@ -0,0 +1,48 @@
+import { createContext, useEffect, useState } from "react";
+import { JobDifficulty, Tag } from "src/models";
+import { getItems } from "src/services/common.service";
+
+export const CommonsContext = createContext<any>({});
+
+export const CommonsProvider = props => {
+    const [tags, setTags] = useState<Array<Tag>>([]);
+    const [categories, setCategories] = useState<Array<Tag>>([]);
```

Comment:
ça fonctionne pas de juste mettre`<Tag[]>` ?

```suggestion
    const [tags, setTags] = useState<Tag[]>([]);
    const [categories, setCategories] = useState<Tag[]>([]);
```

Reply:
carré